llvm/test/Transforms/LoopVectorize/RISCV/evl-compatible-loops.ll

```diff
@@ -0,0 +1,103 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 4
+; RUN: opt -passes=loop-vectorize -force-tail-folding-style=data-with-evl \
```

Comment:
please add the tests in a separate commit/PR, so the diff shows the changed by this patch only

Reply:
Created a separate PR at https://github.com/llvm/llvm-project/pull/92203